src/ast/nodes/CallExpression.ts

```diff
@@ -1,14 +1,17 @@
 import CallOptions from '../CallOptions';
 import ExecutionPathOptions from '../ExecutionPathOptions';
 import SpreadElement from './SpreadElement';
-import { isGlobalVariable } from '../variables/GlobalVariable';
```

Comment:
There seems to be one other use of this in TaggedTemplateExpression. If doing both, that actually removes all usages of this function entirely as well, although can be left to tree shaking too :)

Reply:
Removed